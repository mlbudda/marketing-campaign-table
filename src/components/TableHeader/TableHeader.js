const TableHeader = () => {
  return (
    <tr>
      <th>Name</th>
      <th>User Name</th>
      <th>Start Date</th>
      <th>End Date</th>
      <th>Active</th>
      <th>Budget</th>
    </tr>
  );
};

export default TableHeader;
