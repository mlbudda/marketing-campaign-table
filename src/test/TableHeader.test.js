import React from "react";
import renderer from "react-test-renderer";
import TableHeader from "../components/TableHeader";

it("renders correctly", () => {
  const tree = renderer.create(<TableHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
