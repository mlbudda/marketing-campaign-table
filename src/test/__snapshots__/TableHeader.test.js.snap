// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tr>
  <th>
    Name
  </th>
  <th>
    User Name
  </th>
  <th>
    Start Date
  </th>
  <th>
    End Date
  </th>
  <th>
    Active
  </th>
  <th>
    Budget
  </th>
</tr>
`;
